micropython/drivers/storage/sdcard/sdcard.py

```diff
@@ -63,89 +77,89 @@ def init_spi(self, baudrate):
             # on pyboard
             self.spi.init(master, baudrate=baudrate, phase=0, polarity=0)
 
+    @staticmethod
+    def gb(bigval, b0, bn):
```

Comment:
I suggest making this a global function in the module, ie not part of the `SDCard` class.  Also name it `_gb` to indicate it's a private function.